mitosheet/src/mito/components/taskpanes/Download/DownloadTaskpane.tsx

```diff
@@ -199,16 +204,33 @@ const DownloadTaskpane = (props: DownloadTaskpaneProps): JSX.Element => {
                         </Select>
                     </Row>
                     { props.uiState.exportConfiguration.exportType === 'excel' && 
-                        <ExcelDownloadConfigSection 
-                            dfNames={props.dfNames}
-                            mitoAPI={props.mitoAPI}
-                            userProfile={props.userProfile}
-                            sheetDataArray={props.sheetDataArray}
-                            exportState={props.uiState.exportConfiguration as ExcelExportState}
-                            setUIState={props.setUIState}
-                            newlyFormattedColumns={newlyFormattedColumns}
-                            setNewlyFormattedColumns={setNewlyFormattedColumns}
-                        />  
+                        <div>
+                            <Row justify='space-between' align='center'>
+                                <Col style={{ display: 'flex' }}>
+                                    <p className="text-header-3">Export with formatting</p>&nbsp;
+                                    <ProIcon/>
```

Comment:
Same as above -- let's only display this in the case that the user is not on Pro.